deployment/api-deployment.yaml

```diff
@@ -0,0 +1,57 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  annotations:
+    kompose.cmd: kompose convert -f ../docker-compose.yml.dev
```

Comment:
why don't we use the docker-compose.yml.img as source ?

Reply:
This annotation should be removed

Reply:
This was chosen as the build target was to staging and not production, and as such should be as similar to stage (dev) as possible?

I will remove these today